web/components/input/buttons/CheckboxGroup/index.js

```diff
@@ -11,18 +11,18 @@ const Checkbox = ({ name, value, label, className, checked, disabled, onChange }
         const contextOnChange = event => context.onChange(event.target.value);
         return (
           <div className={`checkbox user-select-none margin-bottom-s margin-left-s ${className}`}>
```

Comment:
can we replace `margin-left-s` with `margin-right-s` here? In the design, there's no margin to the left of the left most checkbox

Reply:
Done